src/preset_cli/cli/superset/sync/dbt/metrics.py

```diff
@@ -89,32 +89,27 @@ def get_metric_expression(metric_name: str, metrics: Dict[str, MetricSchema]) ->
         return f"COUNT(DISTINCT {sql})"
 
     if type_ in {"expression", "derived"}:
+        if metric.get("skip_parsing"):
+            return remove_line_break_wrappers(sql)
```

Comment:
Thought it made more sense to handle this `skip_parsing` scenario here (as part of the ``get_metric_expression`` method than to avoid calling it altogether. This is also useful to ensure that recursive calls to this method handle this logic properly.